### PR TITLE
feat(ios): added all the required empty methods to inherit them at required place.

### PR DESCRIPTION
**Pull request by anil-shukla-axway**
https://jira.appcelerator.org/browse/MOD-2835

### CODE REVIEW

**Comment by anil-shukla-axway** [comment]
@satinder-singh-axway  @fasihul-axway  Please review this PR.

**Review by satinder-singh-axway — CHANGES_REQUESTED** [review]

**Comment by satinder-singh-axway** [comment]
@anil-shukla-axway , can you please address build warnings ?

File: ios/Classes/TagTechnologyProxy.m [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+}
+
+- (void)restartPolling:(id)unused
+{
+}
+
+- (NSNumber *)isConnected:(id)unused
+{
+}
```
[the remark this answers is not in the document]
Comment (anil-shukla-axway) [reply ios/Classes/TagTechnologyProxy.m]: Yes, I have added the same.

File: ios/Classes/TagTechnologyProxy.m [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- (void)connectedToTag:(id)unused
+{
+}
+
+- (NSNumber *)isEnabled:(id)unused
+{
```
[the remark this answers is not in the document]
Comment (anil-shukla-axway) [reply ios/Classes/TagTechnologyProxy.m:35]: added

[the remark this answers is not in the document]
Comment (satinder-singh-axway) [reply ios/Classes/TagTechnologyProxy.m:35]: Where is the ret value value ?

**Review by satinder-singh-axway — CHANGES_REQUESTED** [review]
Please update the PR to merge against a feature branch instead the master.

File: ios/Classes/TagTechnologyProxy.m [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+- (void)restartPolling:(id)unused
+{
+}
+
+- (NSNumber *)isConnected:(id)unused
+{
+  return;
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.m:31]:
```suggestion
  return [NSNumber numberWithBool:NO];
```

File: ios/Classes/TagTechnologyProxy.m [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+- (NSNumber *)isEnabled:(id)unused
+{
+  return;
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.m:36]:
```suggestion
  return [NSNumber numberWithBool:NO];
```

**Review by vikas-goyal-axway — CHANGES_REQUESTED** [review]
Screenshot 2020-12-17 at 12 11 34 PM
@anil-shukla-axway , Also there is 2 build warning, Please look into it

File: ios/Classes/TagTechnologyProxy.m (hunk 2 above)
[the remark this answers is not in the document]
Comment (anil-shukla-axway) [reply ios/Classes/TagTechnologyProxy.m:35]: now updated as per suggestion

File: ios/Classes/TagTechnologyProxy.m (hunk 4 above)
Comment (anil-shukla-axway) [reply ios/Classes/TagTechnologyProxy.m]: Updated as per suggestion

File: ios/Classes/TagTechnologyProxy.m (hunk 3 above)
Comment (anil-shukla-axway) [reply ios/Classes/TagTechnologyProxy.m]: Updated as per suggestion

**Comment by anil-shukla-axway** [comment]
Updated base branch from 'master' to 'NativeTagSupport'

File: ios/Classes/TagTechnologyProxy.h [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+#import "TiProxy.h"
+#import <CoreNFC/CoreNFC.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface TagTechnology : TiProxy
+
+- (void)connect:(id)unused;
+
+- (void)close:(id)unused;
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.h:17]: remove it if this function is not required

File: ios/Classes/TagTechnologyProxy.h [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+- (void)restartPolling:(id)unused;
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.h:19]: remove it if this function is not required

File: ios/Classes/TagTechnologyProxy.h [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+- (NSNumber *)isEnabled:(id)unused;
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.h:21]: remove it if this function is not required

File: ios/Classes/TagTechnologyProxy.m [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+- (void)connect:(id)unused
+{
+}
+
+- (void)close:(id)unused
+{
+}
+
+- (void)restartPolling:(id)unused
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.m:25]: remove it if this function is not required

File: ios/Classes/TagTechnologyProxy.m [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+@implementation TagTechnology
+
+#pragma mark Public API's
+
+- (void)connect:(id)unused
+{
+}
+
+- (void)close:(id)unused
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.m:21]: remove it if this function is not required

File: ios/Classes/TagTechnologyProxy.m [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+- (void)restartPolling:(id)unused
+{
+}
+
+- (NSNumber *)isConnected:(id)unused
+{
+  return [NSNumber numberWithBool:NO];
+}
+
+- (NSNumber *)isEnabled:(id)unused
```
Comment (vikas-goyal-axway) [inline ios/Classes/TagTechnologyProxy.m:34]: remove it if this function is not required

**Review by vikas-goyal-axway — CHANGES_REQUESTED** [review]

**Review by vikas-goyal-axway — APPROVED** [review]

**Pull request merged**